src/views/blocks/tickets/submit-button.php

```diff
@@ -11,14 +11,27 @@
  * @link {INSERT_ARTICLE_LINK_HERE}
  *
  * @since 4.9.3
+ *
  * @version 4.9.4
  *
  */
-
 ?>
 <button
 	class="tribe-block__tickets__buy"
 	type="submit"
 >
 	<?php esc_html_e( 'Add to cart', 'event-tickets' ); ?>
 </button>
+<?php
```

Comment:
this is temporary add to get basic functionality, another ticket I believe handles the final coding.

Reply:
this was removed and put into its own template already.